code/game/antagonist/antagonist_panel.dm

```diff
@@ -27,13 +27,13 @@
 		var/mob/M = player.current
 		dat += "<tr>"
 		if(M)
-			dat += "<td><a href='?src=\ref[src];adminplayeropts=\ref[M]'>[M.real_name]</a>"
+			dat += "<td><a href='?src=\ref[src];adminplayeropts=\ref[M]'>[M.real_name]/([player.key])</a>"
```

Comment:
I noted on Bay that this line was intended to give access to the player panel, the issue is that the href section is incorrect.